### ISSUE TITLE
rollup not found

### ISSUE THREAD

**Issue by sbenthall** [yrevocnu/psychedelic-glass-bead-game#3]
>>$ cd web/
$ yarn dev
yarn run v1.21.1
warning package.json: No license field
$ rollup -c -w
/bin/sh: 1: rollup: not found
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
>>

**Comment by sbenthall** [comment]
I could get past this using `yarn install` after `cd web`

**Comment by benheller** [comment]
Yes! You need to `yarn install` in both the server directory and the web directory (since web is a separate project that just happens to be in this repo). We should discuss whether to pull that out as a separate repository. I'm in favor, they were only combined due to laziness / weekend-hackathon constraints. Thoughts @sbenthall?
